02-oop-inheritance/javaScript/downloableMovie.js

```diff
@@ -0,0 +1,59 @@
+function MovieObserver() {
+
+  this.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+  } 
+  this.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+  } 
+}
+
+function Movie() {
+  this.atributes = new Array();
+  this.observer = new MovieObserver();
+}//end constructor
+
+Movie.prototype = {
+  constructor: Movie,
+  get: function(atribute) {
+     return this.atributes[atribute];
+  }, //end get
+
+  set:function(atribute, value) {
+    this.atributes[atribute] = value;
```

Comment:
Same here, I think this is not working. Also if I set this like sarasaMovie.set(10) from the begging this will break